plugins/outputs/kinesis/kinesis_test.go

```diff
@@ -20,6 +20,7 @@ func TestPartitionKey(t *testing.T) {
 	testPoint := testutil.TestMetric(1)
 
 	k := KinesisOutput{
+		Log: testutil.Logger{},
```

Comment:
Any opposition to me adding log line tracking to `testutil.Logger` so that I can assert the logged lines for particular test cases?